src/GraphQl/Serializer/ItemNormalizer.php

```diff
@@ -23,23 +23,32 @@
 
 use ApiPlatform\Core\Metadata\Property\PropertyMetadata;
 use ApiPlatform\Core\Serializer\AbstractItemNormalizer;
+use ApiPlatform\Core\Serializer\ItemNormalizer as GenericItemNormalizer;
```

Comment:
Can you use `BaseItemNormalizer` as alias please (we usually follow this convention).